src/components/data-management/ProjectDetails.jsx

```diff
@@ -0,0 +1,279 @@
+import {
+  Table, Typography, Space, Tooltip, PageHeader, Button, Input, Descriptions,
+} from 'antd';
+import React, { useState, useEffect } from 'react';
+import { ReloadOutlined, UploadOutlined, EditOutlined } from '@ant-design/icons';
+import _ from 'lodash';
+import PropTypes from 'prop-types';
+import useSWR from 'swr';
+import SpeciesSelector from './SpeciesSelector';
+import MetadataEditor from './MetadataEditor';
+import EditableField from '../EditableField';
+import FileUploadModal from './FileUploadModal';
+import getFromApiExpectOK from '../../utils/getFromApiExpectOK';
+
+const { Text } = Typography;
+
+const ProjectDetails = ({ width, height }) => {
+  const [uploadModalVisible, setUploadModalVisible] = useState(false);
+
+  const { data: speciesData } = useSWR(
+    'https://biit.cs.ut.ee/gprofiler/api/util/organisms_list/',
+    getFromApiExpectOK,
+  );
+
+  const [data, setData] = useState([]);
+  const [sortedSpeciesData, setSortedSpeciesData] = useState([]);
+
+  useEffect(() => {
+    if (!speciesData) {
+      return;
+    }
+
+    const commonSpecies = ['hsapiens', 'mmusculus', 'drerio', 'ggallus'];
```

Comment:
does that mean that if we have user with data that is not from any of these species, we can't support them?

Reply:
No, this just ensures that those species will always be at the top of the list of species in that order, as set out in the wireframes.

Reply:
ah, ok